### PR TITLE
Menu Icons adapted and adding Translations

### PR DESCRIPTION
**Pull request by henryk86**

### CODE REVIEW

**Comment by henryk86** [comment]
This is my first time working with react... so please be very critical ;)

**Comment by henryk86** [comment]
Is there a chance to get the teddycloud version? I Would love to show the current version somewhere like in the old ui...

**Comment by SciLor** [comment]
> Is there a chance to get the teddycloud version? I Would love to show the current version somewhere like in the old ui...
/api/get/internal.version.v_long

To easily get ideas of the API check the network requests. With the way above you can access all settings and internal variable of teddyCloud
https://github.com/toniebox-reverse-engineering/teddycloud/blob/master/src/settings.c

for other endpoints you can check 
https://github.com/toniebox-reverse-engineering/teddycloud/blob/3f301e097afa9a3ae5c243f7835a169e11bccdae/src/server.c#L71

**Comment by SciLor** [comment]
> This is my first time working with react... so please be very critical ;)

It would be great to give some more indiciations what you have changed. Picture would be helpful, so I can see the changes without spinning up a npm with your draft.

**Comment by henryk86** [comment]
> > This is my first time working with react... so please be very critical ;)
> 
> It would be great to give some more indiciations what you have changed. Picture would be helpful, so I can see the changes without spinning up a npm with your draft.

Icons:

image

image

Translations:

The other changes are mostly only use translation functionality instead of hardcoded strings.

File: public/translations/de.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
     },
     "certificates": {
       "navigationTitle": "Zertifikate",
-      "title": "Zertifikate"
+      "title": "Zertifikate",
+      "uploadText" : "Klicken oder ziehen Sie die Datei zum Hochladen in diesen Bereich",
```
Comment (SciLor) [inline public/translations/de.json:25]: Sie / du gemischt verwendet. Ich denke alles auf du macht Sinn.

**Comment by SciLor** [comment]
Perfect =)

**Review by CTeipen — APPROVED** [review]

**Comment by SciLor** [comment]
Nice one. Could you also add a link to the wiki and telegram? Maybe as a list in the end, including the forum link?

**Comment by henryk86** [comment]
So. added the links and simplified the displayed cards:

image

I am sure, there is a better way than duplicating the tonieCardList and tonieCard files to hide some stuff... feel free to improve

**Comment by henryk86** [comment]
Habe da mal noch ne width beim Menu angegeben... irgendwie wurde egal wie breit das fenster war nur eins der drei Menüeinträge angezeigt...

**Comment by henryk86** [comment]
soooo. done. i stop looking around in the code. no more changes from my side here. please merge ;)

**Pull request merged**